### PR TITLE
 Support httpapi in resource modules, part 2

### PR DESCRIPTION
**Pull request by Qalthos**
Depends-On: #29
##### SUMMARY

Modifies the tests for several modules

See also #28

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

eos_acls
eos_acl_interfaces
eos_l2_interfaces
eos_l3_interfaces

### CODE REVIEW

File: tests/integration/targets/eos_l3_interfaces/tasks/main.yaml [hunk 1]
```diff
@@ -1,17 +1,8 @@
 ---
-- name: collect all cli test cases
-  find:
-    paths: '{{ role_path }}/tests/cli'
-    patterns: '{{ testcase }}.yaml'
-  register: test_cases
-  delegate_to: localhost
+- include: cli.yaml
+  tags:
+    - cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_l3_interfaces/tasks/main.yaml:4]: network_cli

File: tests/integration/targets/eos_l3_interfaces/tasks/main.yaml [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-
-- name: run test cases (connection=ansible.netcommon.network_cli)
-  include: '{{ test_case_to_run }} ansible_connection=ansible.netcommon.network_cli'
-  with_items: '{{ test_items }}'
-  loop_control:
-    loop_var: test_case_to_run
-  tags: connection_network_cli
+- include: eapi.yaml
+  tags:
+    - eapi
```
Comment (pabelanger) [inline tests/integration/targets/eos_l3_interfaces/tasks/main.yaml:8]: httpapi

**Comment by Qalthos** [comment]
@pabelanger is this what you wanted?

File: tests/integration/targets/eos_acl_interfaces/tasks/main.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 - include: cli.yaml
   tags:
-    - cli
+    - connection_network_cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_acl_interfaces/tasks/main.yaml:4]: we can drop 
```
connection_
```
on all changes.  Just network_cli or httpapi

**Pull request merged**